web/src/components/InfiniteScrollList.tsx

```diff
@@ -0,0 +1,91 @@
+import { isEmpty } from 'lodash'
```

Comment:
Most of the logic moved from `TuNewsPage` and `TuNewsList`. Please also test tunews.